src/main/java/org/terasology/additionalRails/action/RailSwitchAction.java

```diff
@@ -1,74 +1,167 @@
 package org.terasology.additionalRails.action;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.terasology.additionalRails.components.RailSwitchComponent;
 import org.terasology.entitySystem.entity.EntityRef;
 import org.terasology.entitySystem.entity.lifecycleEvents.OnChangedComponent;
+import org.terasology.entitySystem.event.Event;
 import org.terasology.entitySystem.event.ReceiveEvent;
-import org.terasology.entitySystem.prefab.internal.PrefabFormat;
 import org.terasology.entitySystem.systems.BaseComponentSystem;
 import org.terasology.entitySystem.systems.RegisterMode;
 import org.terasology.entitySystem.systems.RegisterSystem;
+import org.terasology.logic.common.ActivateEvent;
+import org.terasology.logic.console.Console;
+import org.terasology.math.SideBitFlag;
+import org.terasology.minecarts.blocks.RailComponent;
+import org.terasology.minecarts.blocks.RailsUpdateFamily;
 import org.terasology.registry.CoreRegistry;
 import org.terasology.registry.In;
-import org.terasology.signalling.components.SignalConsumerComponent;
 import org.terasology.signalling.components.SignalConsumerStatusComponent;
 import org.terasology.world.BlockEntityRegistry;
 import org.terasology.world.WorldProvider;
 import org.terasology.world.block.Block;
 import org.terasology.world.block.BlockComponent;
 import org.terasology.math.geom.Vector3i;
 import org.terasology.world.block.BlockManager;
+import org.terasology.world.block.family.BlockFamily;
```

Comment:
This and some other imports here and there in affected files appear to not be used. If you spot those (even if you didn't introduce them) please fix by simply removing them :-) A good IDE will show which are unused.